### ISSUE TITLE
Integrate new options to enable ESLint config to Vaadin and Hilla CLI

### ISSUE THREAD

**Issue by web-padawan** [vaadin/eslint-config-vaadin#44]
## Description

Depends on #43

Once we have a beta / stable release of the config, we should consider adding it to new projects:

1. Add option to enable ESLint config to [`@vaadin/cli`](https://github.com/vaadin/cli) (could ask "enable Lit templates?"),
2. Add option to enable ESLint config to [`@hilla/cli`](https://github.com/vaadin/hilla-cli) (with a proper preset for Lit / React),
3. Add a checkbox to https://start.vaadin.com equivalent to the corresponding CLI flag.

Then we could also document TS style guide in Vaadin / Hilla - I'll create a separate issue for it.
